stellar-dotnet-sdk-test/TransactionTest.cs

```diff
@@ -282,6 +282,29 @@ public void TestSha256HashSigning()
             Assert.IsTrue(transaction.Signatures[0].Hint.InnerValue.SequenceEqual(rangeHashCopy));
         }
 
+        [TestMethod]
```

Comment:
There doesn't seem to be any `TransactionBaseTest` class, so I put in the bare minimum test case here.

I don't want to bloat `TransactionTest` with possible code paths for these functions